lib/errors.js

```diff
@@ -161,8 +161,8 @@ class PreconditionFailed extends HttpClientError {
 exports.PreconditionFailed = PreconditionFailed;
 
 class UnprocessableEntity extends HttpClientError {
-  constructor(message) {
-    super(422, 'Unprocessable Entity', message);
+  constructor(message, reason) {
+    super(422, reason || 'Unprocessable Entity', message);
```

Comment:
422 --> if you can use it constant, better. Not a must fix!!

Reply:
Done!